evm/its.js

```diff
@@ -666,14 +686,23 @@ async function processCommand(_axelar, chain, chains, action, options) {
 
         case 'link-token': {
             const [tokenId, destinationChain, destinationTokenAddress, type, operator] = args;
-            const { gasValue } = options;
+            const { gasValue, env } = options;
             const deploymentSalt = getDeploymentSalt(options);
 
+            const submittedGasValue = !gasValue
+                ? await calculateItsCrossChainGas({
+                      sourceChain: chain.axelarId,
+                      destinationChain,
+                      env,
+                      eventType: 'LinkToken',
+                  })
+                : gasValue;
```

Comment:
```suggestion
            const { env } = options;
            const deploymentSalt = getDeploymentSalt(options);

            const gasValue = options.gasValue
                || await calculateItsCrossChainGas({
                      sourceChain: chain.axelarId,
                      destinationChain,
                      env,
                      eventType: 'LinkToken',
                  });
```
more concise using `||`, less intermediate variables